l10n_es_pos/static/src/xml/pos.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Para que salga el ticket correctamente hay que hacer algún cambio y adecuar a las nuevas herencias.
No tengo claro que esta tenga que ser la forma, pero aplicando estos cambios y metiendo en la función `export_for_printing` de `models.Order` otros cambios que comento en `models.js` sale correctamente:

    <t t-name="OrderReceipt" t-inherit="point_of_sale.OrderReceipt" t-inherit-mode="extension" owl="1">
        <xpath expr="//t[@t-if='receipt.company.contact_address']" position="before">
            <t t-if="env.pos.config.iface_l10n_es_simplified_invoice and !receipt.is_to_invoice">
                <div><t t-esc='receipt.date.localestring' /></div>
                <div>Simplified invoice></div>
                <div><t t-esc="receipt.simplified_invoice" /></div><br />
                <t t-log="receipt"/>
            </t>
        </xpath>
        <xpath expr='//t[@t-if="receipt.company.vat"]' position="after">
            <div t-if="receipt.company.street"><t t-esc="receipt.company.street"/></div>
            <div>
                <t t-if="receipt.company.zip"><t t-esc="receipt.company.zip"/> </t>
                <t t-if="receipt.company.city"><t t-esc="receipt.company.city"/> </t>
                <t t-if="receipt.company.state_id">(<t t-esc="receipt.company.state_id[1]"/>)</t>
            </div>
        </xpath>
        <xpath expr='//t[@t-if="receipt.cashier"]' position="after">
            <br />
            <t t-if="env.pos.get_client()">
                <div>Customer: <t t-esc="env.pos.get_client().name" /></div>
                <t t-if="env.pos.get_client().vat">
                    <div><t t-esc="receipt.company.vat_label"/>:<t t-esc="env.pos.get_client().vat" /></div>
                </t>
                <t t-if="env.pos.get_client().address != ',  , '">
                    <div><t t-esc="env.pos.get_client().address" /></div>
                </t>
            </t>
        </xpath>
        <xpath expr="//t[@t-esc='tax.name']" position="after">
            <span> on <t t-esc="env.pos.format_currency(tax.base)" /></span>
        </xpath>
        <xpath expr="//div[hasclass('pos-receipt-order-data')]" position="attributes">
            <attribute name="t-if">!env.pos.config.iface_l10n_es_simplified_invoice or receipt.is_to_invoice</attribute>
        </xpath>
    </t>